OpenTelemetry.AutoInstrumentation.nuspec

```diff
@@ -0,0 +1,66 @@
+﻿<?xml version="1.0"?>
+<package>
+  <metadata>
+    <id>OpenTelemetry.AutoInstrumentation</id>
+    <version>0.3.1-beta.1</version>
+    <description>OpenTelemetry Auto-Instrumentation</description>
+    <authors>OpenTelemetry Authors</authors>
+    <owners>OpenTelemetry Authors</owners>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <license type="expression">Apache-2.0</license>
+    <projectUrl>https://opentelemetry.io/</projectUrl>
+    <tags>Observability OpenTelemetry Monitoring Telemetry Tracing Auto Automatic Instrumentation</tags>
+    <repository type="git" url="https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation" />
+    <dependencies>
+      <group targetFramework="netcoreapp3.1">
+        <dependency id="OpenTelemetry" version="1.3.1" />
```

Comment:
Not for the initial version but any idea to ensure that the list stays up-to-date?

Reply:
I guess we can generate these using custom nuke target.